### DISCUSSION TITLE
Debug statements

### DISCUSSION THREAD

**Discussion by usebeforefree** [Raiden1411/zabi#128] [Ideas]
From my experience the debug statements, when interacting with clients, are way too long and just consume too much space in my terminal window. 

`                ex. httplog.debug("Got response from server: {s}", .{res_body});
`

They obviously go away when you build release, but still, from my perspective they are way too long, and not particularly useful while developing.

Maybe we could pass a slice of lets say first 100 characters, so the statement fits in one line.

Not sure what is the general opinion on this topic, but I am willing to fix the issue myself if it comes to that.

Cheers!

**Comment by Raiden1411** [comment]
I would be okay with that :)

Feel free to make a pr or I can make it later

**Reply by usebeforefree** [reply]
Nice, I'll do it.
